### DISCUSSION TITLE
Command line: Search using "Variable modifications UniMod:1830"

### DISCUSSION THREAD

**Discussion by ba-tata** [vdemichev/DiaNN#388] [Q&A]
Hello ,
I'd like to use UniMod: 1830 as a variable modification to analyze DIA data (biotin-phenol modified sample: APEX2 proteome data).
The other modifications are Fixed: Carbamidometyl (C), Var: Oxidation (M), Acetyl (N-term).
When I searched the DIA data using the above modifications, no peptides with UniMod:1830 were identified. I think this is quite weird.
Could you tell me the mistake of the following parameters.

[DIA-NN 1.8 (Data-Independent Acquisition by Neural Networks)]

./diann.exe --f D:/TempRawFile/220518-Kojima-APEX2-biotinphenol/211216mi-12-kojima-293T-V5-APEX2+crRNA-200ng-DIA_Slot1-02_1_7078.d --lib "" --threads 48 --verbose 2 --out "D:/diaNN/Result/biotinTest/report.tsv" --qvalue 0.5 --matrices  --temp "D:/diaNN/Result/biotinTest" --out-lib "D:/diaNN/Result/biotinTest/report-lib.tsv" --gen-spec-lib --predictor --fasta "C:/Users/massUser/Desktop/SwissProt_2020_03/sprot_human_20200318_contami_NAT1-Cas.fasta" --fasta-search --min-fr-mz 200 --max-fr-mz 1800 --met-excision --cut K*,R*,!*P --missed-cleavages 1 --min-pep-len 7 --max-pep-len 30 --min-pr-mz 300 --max-pr-mz 1800 --min-pr-charge 1 --max-pr-charge 4 --unimod4 --var-mods 1 --var-mod UniMod:35,15.994915,M --var-mod UniMod:1,42.010565,*n --monitor-mod UniMod:1 --var-mod UniMod:1830,361.146012,Y --monitor-mod UniMod:1830 --reanalyse --smart-profiling --full-unimod

**Comment by vdemichev** [comment]
Comments: 
- Without --strip-unknown-mods the UniMod:1830 modification is simply ignored - that's why it's not in the output. Please see the log, DIA-NN prints a warning about this
- Please try searching without any other variable modifications and with precursor charge range 2-3
- MBR must not be used with --qvalue setting greater than 0.01, DIA-NN prints a warning about this
- Please use DIA-NN 1.8.1
